fog/view/enclave/src/lib.rs

```diff
@@ -155,6 +155,25 @@ impl ViewEnclaveApi for SgxViewEnclave {
         mc_util_serial::deserialize(&outbuf[..])?
     }
 
+    fn view_store_init(&self, view_store_id: ResponderId) -> Result<ClientAuthRequest> {
```

Comment:
Nit: Keep `client_accept` and `client_close` next to each other.

Reply:
Done.